src/PHPUnit/Util/Log/JSON.php

```diff
@@ -30,17 +46,17 @@ class JSON extends \PHPUnit\Util\Printer implements \PHPUnit\Framework\TestListe
     /**
      * An error occurred.
      *
-     * @param \PHPUnit\Framework\Test $test
-     * @param \Throwable $e
+     * @param TestCase|PHPUnitTest $test
```

Comment:
PHPUnit\Framework\TestCase implements PHPUnit\Framework\Test interface, so it isn't necessary to document TestCase separately.
I think that @param annotation can be removed.

Reply:
`writeCase` method below is typed to receive a TestCase. 
```php
protected function writeCase(string $status, float $time, array $trace = [], string $message = '', ?TestCase $test = null): void
```
This is more of a help to the IDE, if I remove the annotation it would be saying that any object that implements the `Test` interface can call `writeCase` and that is not true:

`Expected parameter of type '\PHPUnit\Framework\TestCase|void', '\PHPUnit\Framework\Test' provided.`

Reply:
I changed it just as you suggested with a co-authored commit.